docker-build/scripts/00_setup.sh

```diff
@@ -9,7 +9,11 @@ STRIP_SYMBOLS=1
 
 MYARCHTUNE=""
 MYCFLAGS=""
-MYRWDIFLAGS="-O3 -g -DNDEBUG"
+MYBUILDTYPE="RELWITHDEBINFO"
+RELWITHDEBINFOFLAGS="-O3 -g -DNDEBUG"
+DEBUGFLAGS="-O3 -g -DDEBUG -D_DEBUG -DNO_CATCH_EXCEPTIONS"
```

Comment:
I would expect `-Og` for debug builds so that compilee doesn't optimize all out making it hard to use with debugger.